docs/GETTING-STARTED.md

```diff
@@ -68,6 +75,74 @@ app.post('/login', function(req, res) {
 });
 ```
 
+As said earlier, API documentation filters could be put in place before having such API rendered on the JSON file. A sample is shown in [app.js](../example/v2/app.js) where some form of filtering is done.
+```javascript
+function jsDocFilter(jsDocComment) {
+    // Do filtering logic here in order to determine whether
+    // the JSDoc under scrunity will be displayed or not.
+    // This function must return boolean. `true` to display, `false` to hide.
+    const docDescription = jsDocComment.description;
+
+    const features = docDescription.indexOf('feature') > -1;
+    const featureX = docDescription.indexOf('featureX') > -1; // featureX is the filter keyword
+    const featureY = docDescription.indexOf('featureY') > -1; // featureY is also another filter keyword
+    
+    // `featureFilter` is some external environment variable
+    const enabledX =
+      featureX && envVars && envVars.featureFilter.indexOf('X') > -1;
+    const enabledY =
+      featureY && envVars && envVars.featureFilter.indexOf('Y') > -1;
+
+    const featuresEnabled = enabledX || enabledY;
+
+    const existingRoutes = [];
+
+    function includeDocs() {
+      const route =
+        jsDocComment &&
+        jsDocComment.tags &&
+        jsDocComment.tags[0] &&
+        jsDocComment.tags[0].description &&
+        jsDocComment.tags[0].description.split(':')[0];
+
+      if (existingRoutes.indexOf(route) === -1) {
+        // need to perform check if the route doc was previously added
+        return true;
+      }
+
+      return false;
+    }
+
+    // featured route documentation
+    if (features) {
```

Comment:
This is the place where I'd change the logic to return a reduced version of the docs.